src/blog/2023/01/getting-started-with-node-red.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting Started with Node-RED
+subtitle: Node-RED is one of the easiest ways to program ever created but everyone needs a little help
+description: In this article we are going to explain the first things we think you need to know to get started with Node-RED
+date: 2023-01-23
+authors: ["rob-marcer"]
+---
+
+
+Node-RED is a visual programming tool for working with IoT devices and web services. It allows users to create flows using a drag-and-drop interface, making it easy to connect different nodes together to build powerful automations.
+<!--more-->
+In this blog post, we'll take a look at how to get started with Node-RED and create some basic flows. Additionally, we will explore the use of the palette manager, a powerful feature that allows users to install and manage additional nodes for Node-RED.
+
+### Installing Node-RED
+
+First, you'll need to install Node-RED on your machine. The easiest way to do this is by using npm (Node Package Manager), which comes with Node.js. To install Node-RED, open up your terminal and type the following command:
```

Comment:
I think we should talk about two ways of using Node-RED: 1) Signing into FF Cloud, and 2) Installing it on your local machine.  Of course using FF Cloud is the easiest and quickest way to use Node-RED. 

I would actually list the steps to sign into FF Cloud first and then have a seperate section for install localling.

Reply:
I've added the option of installing Node-RED on FlowForge and made it the first suggestion.